virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc

```diff
@@ -0,0 +1,51 @@
+[id="virt-managing-vm-snapshots"]
+= Managing virtual machine snapshots
+include::modules/virt-document-attributes.adoc[]
+:context: virt-managing-vm-snapshots
+
+toc::[]
+
+You can create and delete virtual machine (VM) snapshots for VMs, whether the VMs are powered off (offline) or on (online). You can only restore to a powered odd (offline) VM. {VirtProductName} supports VM snapshots on:
+
+* Red Hat OpenShift Container Storage
+* Any other storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
+
+[IMPORTANT]
+====
+Online snapshots are not supported for hot-plugged virtual disks.
```

Comment:
Same comment as above.

Reply:
Noted above, this change is in the new file (virt-managing-vm-snapshots) from your previous comment.